examples/common/tracing/TraceDecoder.h

```diff
@@ -44,6 +44,13 @@ class TraceDecoder : public TraceStream
         snprintf(buffer, sizeof(buffer), "    %s\t %s", tag.c_str(), data.c_str());
         ReadString(buffer);
     }
+    void CleanJsonBuf(void) override
```

Comment:
add whitespace above.

Do not use `(void)` just use `()`

Reply:
Fixed